### ISSUE TITLE
Send message upon first connection to server

### ISSUE THREAD

**Issue by skadakar** [tkuester/taky#43]
Would like the ability to have a connection message like FTS has.

**Comment by tkuester** [comment]
I think this is a very useful feature -- but for now I think I'll push this off to the 1.0 release.

**Comment by pinztrek** [comment]
If this feature is added it needs to be able to be disabled. It also the ideal to have some kind of rate limit intelligence as to the last time it sent the message to that uuid. It indeed will trigger spam connections when driving in and out of data connectivity. I have seen this with FTS and now turn off the connection message as it gets very annoying.

**Comment by tkuester** [comment]
> If this feature is added it needs to be able to be disabled.

Absolutely! Taky will always be first, a dumb router. Conveniences and features like this will always be optional, and "opt-in", not something that gets in the way!
